src/main/webapp/resources/js/pages/announcement/components/AnnouncementsPage.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from "react";
+import { PageWrapper } from "../../../components/page/PageWrapper";
+import {
+  getAnnouncements,
+  markAnnouncementRead,
+} from "../../../apis/announcements/announcements";
+import { Avatar, Form, List, notification, Radio, Typography } from "antd";
+import { PriorityFlag } from "./PriorityFlag";
+import { fromNow } from "../../../utilities/date-utilities";
+import ViewReadAnnouncement from "./ViewReadAnnouncement";
+import ViewUnreadAnnouncement from "./ViewUnreadAnnouncement";
+import { blue1 } from "../../../styles/colors";
+
+const { Text } = Typography;
+
+/**
+ * React component to display the page for administration of users.
+ * @returns {*}
+ * @constructor
+ */
+export function AnnouncementsPage({}) {
+  const [announcements, setAnnouncements] = useState([]);
+  const [readToggle, setReadToggle] = useState(false);
+  const [filter, setFilter] = useState("all");
+
+  useEffect(() => {
+    getAnnouncements().then((data) => {
+      switch (filter) {
+        case "read":
+          return setAnnouncements(
+            data.data.filter((item) => item.announcementUserJoin !== null)
+          );
+        case "unread":
+          return setAnnouncements(
+            data.data.filter((item) => item.announcementUserJoin === null)
+          );
+        default:
+          return setAnnouncements(data.data);
+      }
+    });
+  }, [filter, readToggle]);
+
+  function markAnnouncementAsRead(aID) {
+    return markAnnouncementRead({ aID })
+      .then(() => {
+        setReadToggle(!readToggle);
+      })
+      .catch(({ message }) => {
+        notification.error({ message });
+      });
+  }
+
+  return (
+    <PageWrapper title="Announcements">
+      <Form>
+        <Form.Item>
+          <Radio.Group
+            value={filter}
+            onChange={(e) => setFilter(e.target.value)}
+          >
+            <Radio.Button value="all">All</Radio.Button>
+            <Radio.Button value="unread">Unread</Radio.Button>
+            <Radio.Button value="read">Read</Radio.Button>
+          </Radio.Group>
+        </Form.Item>
+      </Form>
+      <List
+        dataSource={announcements}
+        renderItem={(item) => (
+          <List.Item
+            className="t-announcement-item"
+            style={
+              item.read ? { backgroundColor: blue1, fontWeight: "bold" } : {}
+            }
+          >
+            <List.Item.Meta
+              avatar=<Avatar
+                style={
+                  item.read
+                    ? { backgroundColor: blue1 }
+                    : { backgroundColor: "#fff" }
```

Comment:
Can you only apply the blue background here, and only set the title to bold (can you use `<Text strong>{text</Text>` to keep consistency.  I don't like the date being bolded as well

Reply:
Please see 00b4a5cee25e41d51f70dd1a0f13aea9477824b9.